wds-log-post.php

```diff
@@ -523,11 +532,19 @@ public static function abort_progress( $log_post_id ) {
  * @since  0.1.0
  * @return WDS_Log_Post  Singleton instance of plugin class.
  */
-function wds_log_post() {
-	return WDS_Log_Post::get_instance();
+function wds_log_post( $posttype, $name ) {
+	return WDS_Log_Post::get_instance( $posttype, $name );
 }
 
+add_action( 'plugins_loaded', 'load_wds_log', 9999);
+function load_wds_log() {
 // Kick it off
-if ( apply_filters( 'wds_log_post_site_check', 'is_main_site' ) ) {
-	add_action( 'plugins_loaded', array( wds_log_post(), 'hooks' ) );
+    if ( apply_filters( 'wds_log_post_site_check', 'is_main_site' ) ) {
+        $posttypes = apply_filters( 'wds_log_post_types', array() );
```

Comment:
I was thinking of this problem but in case will see that default post type also if that library is required in plugins that not use that post type.
So the simpliest way for me is when the developer update the lbirary have only to add the new filter and the support of the post type that use.